packages/app-store/routing-forms/components/getServerSidePropsSingleForm.ts

```diff
@@ -30,16 +34,6 @@ export const getServerSidePropsForSingleFormView = async function getServerSideP
     };
   }
 
-  const isFormCreateEditAllowed = (await import("../lib/isFormCreateEditAllowed")).isFormCreateEditAllowed;
-  if (!(await isFormCreateEditAllowed({ userId: user.id, formId, targetTeamId: null }))) {
-    return {
-      redirect: {
-        permanent: false,
-        destination: "/403",
-      },
-    };
-  }
-
```

Comment:
Now that we don't show 404 for a Team Routing Form to a regular member, we should allow the user to navigate to this Routing Form

Right now user can't navigate to it. 
<img width="2964" height="448" alt="image" src="https://github.com/user-attachments/assets/f9d7125b-618a-4f48-baed-90239c61163e" />

With the changes in the PR, save and Delete buttons are correctly disabled
<img width="3300" height="766" alt="image" src="https://github.com/user-attachments/assets/2c551f71-ebb6-4a8d-899a-3065d19be346" />

Reply:
Opened a followup ticket